admin/setup.php

```diff
@@ -88,7 +90,7 @@
 $var=0;
 
 print '<form method="POST" action="'.$_SERVER['PHP_SELF'].'">';
-print '<input type="hidden" name="token" value="'.$_SESSION['newtoken'].'">';
+print '<input type="hidden" name="token" value="'.$newToken.'">';
```

Comment:
Discount rule n'est pas compatible Dolibarr < 12 Donc il faut directement utiliser newToken()
```suggestion
print '<input type="hidden" name="token" value="'.newToken().'">';
```